platforms/js/opencv_js.config.py

```diff
@@ -16,8 +16,9 @@
                 'goodFeaturesToTrack','grabCut','initUndistortRectifyMap', 'integral','integral2', 'isContourConvex', 'line', \
                 'matchShapes', 'matchTemplate','medianBlur', 'minAreaRect', 'minEnclosingCircle', 'moments', 'morphologyEx', \
                 'pointPolygonTest', 'putText','pyrDown','pyrUp','rectangle','remap', 'resize','sepFilter2D','threshold', \
-                'undistort','warpAffine','warpPerspective','warpPolar','watershed', \
-                'fillPoly', 'fillConvexPoly'],
+                'undistort','warpAffine','warpPerspective','warpPolar','watershed', 'Intelligent_scissors',\
```

Comment:
Class should be defined in a different way. See CLAHE, for example. Here are only methods